pitest-entry/src/main/java/org/pitest/coverage/CoverageData.java

```diff
@@ -252,13 +251,7 @@ private TestInfo createTestInfo(final Description description,
   }
 
   private int getNumberOfCoveredLines(final ClassName clazz) {
```

Comment:
The name of the method and the implementation do not fit, what do you think (is not your fault but as I review the code that came to my mind) the method returns theNumberOfCoveredTests(...) not the number of covered lines. Need to check usages of the method later.

Reply:
I think `getTestsForClassName` has wrong name.
Tried to resolve this problem in latest commit.
Please, take a look

Reply:
LGTM